src/script/api/script_airport.cpp

```diff
@@ -155,7 +154,7 @@
 	if (!as->IsWithinMapBounds(0, tile)) return INVALID_TOWN;
 
 	uint dist;
-	return AirportGetNearestTown(as, tile, AirportTileTableIterator(as->table[0], tile), dist)->index;
+	return AirportGetNearestTown(as, DIR_N, tile, AirportTileTableIterator(as->table[0], tile), dist)->index;
```

Comment:
Correct me if I'm wrong. AIs can place rotated airports. They get a list of airport types with each type repeated 4 times, one for reach rotation. Or is that how OpenGFX+ Airports 0.5.0 coded them? What happens now when an AI tries placing an airport of one of the rotated types?

Reply:
Scripts can only build airports using the first layout for a given airport type.
I've adjusted it to use the corresponding rotation.